.gitignore

```diff
@@ -15,3 +15,5 @@ bower.json
 *.log
 *.swp
 *.stackdump
+
+chart.js-*.tgz
```

Comment:
Why is this needed?

Reply:
Similar to the dist folder, when you run `npm pack`, it will generate *chart.js-&lt;version&gt;.tgz* which use for local `npm install`. It may use for testing when the package is not uploaded to npmjs.